src/scss/scss.11ty.js

```diff
@@ -0,0 +1,119 @@
+/* This technique was created by Max Böck in Eleventastic: https://github.com/maxboeck/eleventastic */
```

Comment:
We should record for our own purposes through which other(s) of our own projects we inherited this code - and consider whether it belongs in some library - perhaps trivet?

Reply:
This script is inherited from https://github.com/inclusive-design/wecount.inclusivedesign.ca/. @greatislander probably uses it in other 11ty projects too.

@greatislander, do you think if this script fits in https://github.com/fluid-project/eleventy-plugin-fluid?

Reply:
@cindyli This isn't really a file that can be used in a Netlify plugin as it's essentially a template rather than an Eleventy shortcode or filter. It's not included in Trivet because the feedback I got from @jobara and @jhung was that Trivet should be able to be configured for Sass, PostCSS or Stylus depending on the needs of a project. That might be worth revisiting, though— it seems like we are standardizing around using Sass and since Trivet is meant for new projects it doesn't need to include configurations for all older projects.

More info on this approach here: https://mxb.dev/blog/eleventy-asset-pipeline/

Reply:
Infusion was converted from stylus to Sass. I don't know of any other project that might still be using it, maybe if there is something that hasn't been migrated to 11ty yet. I don't think there are any plans to use Stylus anywhere at the moment.